### PR TITLE
Feature/merge profiles

### PR DESCRIPTION
**Pull request by celestemartinez**
Implements the new function merge_profiles. 

It depends on https://github.com/iesl/openreview/pull/1605.

### CODE REVIEW

File: openreview/openreview.py [hunk 1]
```diff
@@ -475,6 +476,30 @@ def post_profile(self, profile):
         response = self.__handle_response(response)
         return Profile.from_json(response.json())
 
+    def merge_profiles(self, profileTo, profileFrom):
+        """
+        Merges two Profiles
+
+        :param profileTo: Profile object to merge to
+        :type profileTo: Profile
+        :parm profileFrom: Profile object to merge from
```
Comment (mspector) [inline openreview/openreview.py]: looks good. my one suggestion is that we add just a little more description here: maybe something like "Profile to merge from (this profile will be deleted)".

Comment (celestemartinez) [reply openreview/openreview.py]: Fixed

**Comment by melisabok** [comment]
that is true, @celestemartinez could you create a test in test_client?

**Comment by celestemartinez** [comment]
Created test in tes_client

**Pull request merged**